CHANGELOG.rst

```diff
@@ -7,6 +7,15 @@ Change Log
 ----------
 
 
+4.1.0
```

Comment:
The previous PR did not in fact release 4.0.0, it went to master as a beta with version 3.1.0.1b1 in anticipation of this PR being part-two of the 4.0.0 release, so you should merge with the existing 4.0.0 notes, not generate a new version.
```suggestion
4.0.0
```

Reply:
Ah, got it, yeah, that's why I was confused about the version (pyproject not bumped though CHANGES.rst was).